plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -131,7 +131,6 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
 
   private fun createSource(): GeoJsonSource {
     return geoJsonSource(sourceId) {
-      data("")
```

Comment:
This is a breaking change, previously we allow setting empty GeoJsonSource and update the GeoJson afterwards.